Documentation/Signoffs/LaunchingAndFiring.md

```diff
@@ -1,8 +1,8 @@
+This subsystem will use a 12-volt DC brushed motor along with a relay that will use a signal from the aiming subsystem as an input to allow the DC motor to operate. This signal will be sent for the amount of milliseconds that it will take the firing apparatus to fire a round, and return to the preloaded condition. This time will be found once the Mechancial Engineering team has finished constructing their design. Due to the nature of the ME team's design for the firing apparatus, once in preload, the motor will be in a state where there will be no load just before activation of the motor. Once activated by the aiming subsystem's signal the motor will then go through the firing cycle, resetting the interceptor. The final stage of this cycle, when the motor will be returning into the no load section of the firing cycle, will have a margin of error of 15 degrees in either direction (this number is a "safe" number the ME team provided based on their design, it is more likely the acceptable margin of error is higher than 15 degree in either direction but this number they felt safe with using) in which the position of the output shaft can come to rest.(The motor will be geared 30:1, more on this below) During testing and experimentation, the device will be honed in to stop as closely as possible to the most optimal position for the mechanism to function properly by using a proximity sensor and a bolt that will be placed along the rotationary wheel of the design. More on this in an added section below the proof for constraint 5.
```

Comment:
This needs to be rewritten to use the prox. It still reads as if it was time based.